research/10.md

```diff
@@ -45,4 +55,11 @@ Appears to be a message to Operations.
 
 ## Acronyms / Codes
 
+- KHOU - Houston Hobby Airport
```

Comment:
Not necessary to spell these out, here we just need to identify unique codes. I'd say we can just say identify that there are airport codes, etc.

For example maybe:
```
LDR - whatever this means
KXXX - origin and destination airport
```

Reply:
I can update that.